elements/userlevel/jiffieclock.cc

```diff
@@ -61,13 +61,13 @@ JiffieClock::initialize(ErrorHandler*) {
 
 void JiffieClock::run_timer(Timer* t) {
     Timestamp current_time = Timestamp::now_steady();
-    int64_t delta = (current_time - last_jiffies_update).msec() / (1000 / CLICK_HZ);
+    int64_t delta = (current_time - last_jiffies_update).usec() / (1000000 / CLICK_HZ);
```

Comment:
Are you using JiffieClock or did you catch this would break?

Reply:
I just noticed it.